OrbitClientData/ModuleData.cpp

```diff
@@ -75,13 +75,11 @@ void ModuleData::AddSymbols(const orbit_grpc_protos::ModuleSymbols& module_symbo
   is_loaded_ = true;
 }
 
-void ModuleData::ClearSymbols() {
+void ModuleData::UpdateFunctionsModuleBaseAddress(uint64_t module_base_address) {
   absl::MutexLock lock(&mutex_);
-
-  CHECK(is_loaded_);
-  functions_.clear();
-  hash_to_function_map_.clear();
-  is_loaded_ = false;
+  for (const auto& [_, function] : functions_) {
```

Comment:
the line below does not look like it is const....

Reply:
done